crates/bevy_ecs/src/component.rs

```diff
@@ -596,8 +596,10 @@ impl Tick {
     }
 
     #[inline]
-    /// Returns `true` if the tick is older than the system last's run.
-    pub fn is_older_than(&self, last_change_tick: u32, change_tick: u32) -> bool {
+    /// Returns `true` if the tick occurred since the system's `last_change_tick`.
```

Comment:
```suggestion
    /// Returns `true` if the `change_tick` occurred since the system's `last_change_tick`.
```

Reply:
Nevermind, this is wrong.

Reply:
Since there's so many ticks here, I think it would be good to be more specific about what "the tick" refers to (I believe it's `self`).